gulp/FabricBuild.js

```diff
@@ -50,86 +96,48 @@ gulp.task('Fabric-copyAssets', function () {
 // ----------------------------------------------------------------------------
 
 // Build LESS files for core Fabric into LTR and RTL CSS files.
-gulp.task('Fabric-buildLess', function () {
-    var srcPath;
-    var cssPlugin;
-    var fileExtension; 
-    
-    // Check if building SASS
-    if(Config.buildSass) {
-        srcPath = Config.paths.srcSass;
-        cssPlugin = Plugins.sass;
-        fileExtension = Config.sassExtension;
-    } else {
-        srcPath = Config.paths.srcLess;
-        cssPlugin = Plugins.less;
-        fileExtension = Config.lessExtension;
-    }
-    
+gulp.task('Fabric-buildStyles', ['Fabric-configureBuild', 'Fabric-styleHinting'], function () {
```

Comment:
When I put an error into a Less file and build, the console outputs the error 9 times.